mLRS/CommonTx/cli.h

```diff
@@ -290,11 +290,17 @@ class tTxCli
         CLI_STATE_STATS,
     } CLI_STATE_ENUM;
 
+    typedef enum {
+        CLI_OUTPUT_NONE = 0,
+        CLI_OUTPUT_HELP,
+        CLI_OUTPUT_PL,
+    } CLI_OUTPUT_ENUM;
```

Comment:
may be PRINT instead of OUTPUT, as the functions which it manages are also called print